load_gibleed.sh

```diff
@@ -0,0 +1,9 @@
+
+for f in ./data/GiBleed/GiBleed_5.3/CONDITION_OCCURRENCE.csv; do
```

Comment:
the `CONDITION_OCCURRENCE` file is not checked in

if this is a public file, good to add a link to the file here